script/Ciel/Day 4/04_01_CIEL04_1B.txt

```diff
@@ -383,13 +382,13 @@ Today is the last time I will see this sight.
 [sha:0e1a672e0062007497b13f9779ad6e4bb0859879]{
 -- Page 70, Offset 8340.
 -- たった二日だけの里帰りだったが、それを惜しんではいられない。
-// Someone double check this
+// Someone double check this i dont really get this either
```

Comment:
looks right. Though I should note that 里帰り is kinda specific, not just "returning home". It's like the english phrase "moving back in with your parents" here. He's like saying "since I'm back with my family" I guess.